packages/engine/Source/Core/addAll.js

```diff
@@ -0,0 +1,41 @@
+import defined from "./defined.js";
+
+/**
+ * Adds all elements from the given source array to the given target array.
+ *
+ * If the `source` is `undefined`, then nothing will be done. Otherwise,
```

Comment:
```suggestion
 * If the <code>source</code> is <code>undefined</code>, then nothing will be done. Otherwise,
```

I know we're not generating docs for this because it's `@private` but something I always forget about too is that JSDoc comments are not markdown (as much as I wish they were)